integration_tests/dbt_project.yml

```diff
@@ -1,22 +1,22 @@
 name: 'salesforce_integration_tests'
-version: '1.1.1'
+version: '1.2.0'
 config-version: 2
 
 profile: 'integration_tests'
 
 models:
   +materialized: table
-  +schema: "salesforce_{{ var('directed_schema','dev') }}"
+  # +schema: "salesforce_{{ var('directed_schema','dev') }}" --Used for validation tests
 
 vars:
   # enable history models when generating docs!
-  # salesforce__contact_history_enabled: true
-  # salesforce__account_history_enabled: true
-  # salesforce__opportunity_history_enabled: true
+  salesforce__contact_history_enabled: true
+  salesforce__account_history_enabled: true
+  salesforce__opportunity_history_enabled: true
```

Comment:
These should be enabled only when generating the docs. Before merging we need to ensure these are commented out.

Reply:
Thank you for catching, this always trips me up.